src/raster/r.damflood/SWE.c

```diff
@@ -64,17 +81,17 @@ void shallow_water(double **m_h1,double **m_u1, double **m_v1, float **m_z,float
 
 
 
-	// DESCRIPTION OF METHOD (italian --> TRASLATE)
-   	// primo ciclo: calcolo nuove altezze dell'acqua al tempo t+1:
-   	// 		- a valle della diga applico l'equazione di continuita' delle shallow water
-   	// 		  in pratica la nuova altezza e' valutata attraverso un bilancio dei
-   	// 		  flussi in ingresso e in uscita nelle due direzioni principali
-   	// 		- a monte delle diga:
-	//              	- nel metodo 1 e 2 :l'equazione di continuita' e' applicata al volume del lago
-   	//			  fisicamente questo porta a una minore realisticita' ma evita le oscillazioni che
-   	// 			  sono causa di instabilita' numerica
-	//			- nel caso piu' generale si applicano le equazioni a tutto il lago
-
+    // DESCRIPTION OF METHOD
+    // First cycle: Calculation of new water heights at time t + 1:
+    //      - Downstream of the dam: Apply continuity equation to shallow water (?)
+    //        In practice, the new height is evaluated through a balance 
+    //        of the incoming and outgoing flows in the two main directions
+    //      - Upstream of the dam:
+    //          - In methods 1 and 2:
+    //              - The continuity equation is applied to the volume of the lake
+    //                Physically this leads to a less realistic but avoids 
```

Comment:
This didn't make much sense, but it was the closest that kept the same meanings of the Italian comment. Someone has a better explication?

Reply:
I think swe applies where speed in plane directions are of a magnitude higher than in vertical direction, and in a lake it is questionable...